operator_ui/@types/operator_ui/index.d.ts

```diff
@@ -2,7 +2,7 @@ import * as dbTypes from '../db'
 import { status, adapterTypes } from '../constants'
 
 interface RunResult {
-  data: { result: string | null }
+  data: { result: string | undefined }
```

Comment:
https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined

Reply:
`result?`: string